src/lib/settings.js

```diff
@@ -85,6 +96,19 @@ export function toResultsWithRegExp(results) {
 }
 
 /**
+ * Transforms array of possible column with player name headers to RegExp
+ * @param {Array.<string>} columnHeaders
+ * @returns {Array.<RegExp>}
+ */
+export function nameHeadersToRegExp(columnHeaders) {
```

Comment:
Should be given a name of general purpose function - in theory it doesn't have to be used only for headers :)